pam/pam_test/module-transaction-dummy_test.go

```diff
@@ -0,0 +1,512 @@
+package pam_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/msteinert/pam"
+	"github.com/stretchr/testify/require"
+)
+
+func ptrValue[T any](value T) *T {
+	v := value
+	return &v
+}
```

Comment:
Can't we just `return &value`? (Go already passes the function arguments as copies by default)